pkg/action/env.container.go

```diff
@@ -32,6 +32,7 @@ const (
 	containerFlagUseVolumeWD = "use-volume-wd"
 	containerFlagRemoveImage = "remove-image"
 	containerFlagNoCache     = "no-cache"
+	containerFlagEntrypoint  = "entrypoint"
```

Comment:
We also need a flag `--exec` of type bool. Please, read the comment for more info and examples: https://github.com/launchrctl/launchr/issues/29#issuecomment-2028805874

Consider the following:
1. We must disable argument validation on Cobra side
2. We must pass all arguments as is and overwrite `Cmd` with it.

TODO:
- [ ] Implement `--entrypoint` option - not set, set empty string, set some string. Test "not set", if doesn't work, focus on strings only for now.
       I'm not sure "not set" case will work properly with current implementation. May require refactoring of options. **
- [ ] Add a property `ArgsRaw` to struct `Input` and fill it in `CobraImpl`
- [ ] If `--exec` flag is provided, overwrite the `Cmd` in container 
- [ ] Adjust arguments validation to work properly with arguments when `--exec` option is not provided and when we expect Actions exact number of arguments. 
      It is a tricky part. Try to disable validation for now and after we'll adjust.
      You can't directly check `--exec` option in Cobra to change validation of arguments because of loose-coupling with run environment. I have ideas, but they require a lot of refactorings. ***